assets/question_sets/all.js

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Data for the set of all questions.
+ *
+ * If you create a new question, please add it to this list so that the
+ * schema verification tests will run on it before you try to get it
+ * reviewed.
+ */
+
+globalData.questionSets['all'] = {
```

Comment:
Either have a list in the test itself that's all question IDs, or have a list of all question set IDs.